CONTRIBUTING.md

```diff
@@ -42,6 +42,80 @@ If the hook complains and intercepts the commit, you can run PHP-CS-Fixer manual
 
     vendor/bin/php-cs-fixer fix src
 
+We expect contributors to code according to good coding practices also for front-end code. The Mozilla Developer Network has a strong position in the JavaScript/ECMAScript developer community, so we follow their example and use the [Prettier tool](https://prettier.io/docs/en/index.html) to validate the code.
+
+[Writing Style Guide](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#general_guidelines_for_javascript_code_examples)
+
+To test how the Prettier tool works and what the rules are, you can visit the online version [here](https://prettier.io/playground/).
+
+Prettier validates and fixes the code with a pre-commit hook run on git commit. If there are style errors in the code, Prettier automatically corrects them in the files containing the code. If there are syntax errors, Prettier does not fix them, but will show you the errors that must be corrected before the commit can be performed successfully.
+
+If Prettier works unexpectedly, please let us know in the Skosmos [Users group](https://groups.google.com/g/skosmos-users).
+
+#### How to deploy Prettier in your Skosmos installation
+
+(In the same folder where the .git folder is located)
+
+1. 
+```npm install --save-dev --save-exact prettier```
+
+The package.json file will be updated with the new dependency and the node_modules folder will be updated.
+```
+"devDependencies": {
+    "prettier": "2.8.7",
+} 
+```
+[NPM Info](https://www.npmjs.com/package/prettier)
+
+2.
+Create a file prettier.config.js in the root of the Skosmos folder and add the following module:
+```
+module.exports = {
+    trailingComma: 'es5',
+    tabWidth: 4,
+    semi: false,
+    singleQuote: true,
+}
+```
+3.
+Backup .git/hooks/pre-commit-sample and create a file .git/hooks/pre-commit
+
+Write the following script to the file:
+```
+#!/bin/bash
+set -e
+npx prettier --config prettier.config.js --write *.vue *.js
+```
+4.
+To test how Prettier works, the next time you write some JavaScript code, you can deliberately make the code
```

Comment:
I don't think this is necessary for CONTRIBUTING.md. If there is a need for deeper explanations about style, they could go into the wiki and just include a link here.

Reply:
Removed